transport-msg.tex

```diff
@@ -650,38 +650,41 @@ \subsubsection{Device Notifications}
      completion.
 \end{itemize}
 
-These messages enable asynchronous updates from the device, reducing driver
-polling overhead and improving performance.
-
+These messages enable asynchronous updates from the device.
 
 \subsubsection{Configuration Changes During Operation}
 \label{sec:Virtio Transport Options / Virtio Over Messages / Device Operation / ConfigUpdates}
 
-The driver \emph{MAY} issue new \msgref{SET_CONFIG} or \msgref{SET_FEATURES}
-messages after initialization if device functionality or features need to be
-changed at runtime. The device \emph{MUST} respond with updated generation counts
-or feature bits if the request succeeds, or \msgref{ERROR} if the request fails
-(e.g., invalid generation count or unsupported features). The driver \emph{SHOULD}
-be prepared to handle partial acceptance of requested features if the device
-cannot enable them all simultaneously.
+The driver \emph{MAY} issue new \msgref{SET_CONFIG}
+messages after initialization if device configuration needs to be
+changed at runtime. The device \emph{MAY} also make configuration changes at
+runtime and \emph{MUST} signal those changes with \msgref{EVENT_CONFIG}
```

Comment:
I think the "MUST" to signal changes using event_config needs to be lower down to a "SHALL" as we state earlier that there could be cases where this is not done.

Reply:
MUST/SHALL: elsewhere I said MUST send EVENT_CONFIG or EVENT_CONFIG0.  From the Transport level they always get one or the other from the bus.

Reply:
typo missing s??  I don't see it

Reply:
my bad my brain removed the \emph{...}, my comment is not valid

Reply:
I added EVENT_CONFIG0 here and will probably rename that to EVENT_CONFIG_EMPTY later down.